### ISSUE TITLE
Add more problem functions in fitness.py

### ISSUE THREAD

**Issue by Agrover112** [Agrover112/fliscopt#12]
Add more problems rel to current flightscheduling problem. (No matrix problems)
Only problems whose solution is of the form: [1,2,3,4]
OR
[x1,z2..........xn] and corresponding domain : [(-a,b),(-c,-d).......nth-tuple]. Refer fitness.py for existing problems and cost functions and their domains.

Ideally the isse requires you to add a sample cost function to solve a particular problem, whose inputs are of form 1*n

**Comment by Yash621** [comment]
@Agrover112  can i contibute to this ?

**Comment by Agrover112** [comment]
Sure @Yash621  Let ke know what type of problems -funcs/functions you have in mind

**Comment by Agrover112** [comment]
> @Agrover112 can i contibute to this ?

Sorry, I forgot to assign you here you go!

**Comment by Yash621** [comment]
@Agrover112 can you suggest from where i can read about the flight scheduling problem ?

**Comment by Agrover112** [comment]
> @Agrover112 can you suggest from where i can read about the flight scheduling problem ?

Ah, well this ones a bit specific . I myself didn't find a lot , but I'm writing a paper on this .
TLDR, you have people who need to reach a dest from src city and book a flight for to and return flight. So now we need to make sure all people from different cities reach the dest City at roughly the same time with least wait times while arriving and least waiting times while departing ans obviously , minimise the cost in Dollars as well.!

**Comment by Yash621** [comment]
ohh okay ,can you expalin what type of cost functions you have used in this ?

**Comment by Agrover112** [comment]
> ohh okay ,can you expalin what type of cost functions you have used in this ?

The cost function I crewated is specifically for Flight scheduling problem. Rest are benchmark functions for Optimization Algorithms.( For testing performance of algos). Pretty famous you can google their names and find info about them on Wikipedia as well.
